src/app/dashboard/views/DashboardConfigurationForm/DashboardConfigurationSubnetForm/DashboardConfigurationSubnetForm.test.tsx

```diff
@@ -141,22 +115,16 @@ describe("DashboardConfigurationSubnetForm", () => {
       subnet: subnetStateFactory({ items: subnets, loaded: true }),
     });
     const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter>
-          <CompatRouter>
-            <DashboardConfigurationSubnetForm />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
-    );
-    submitFormikForm(wrapper, {
-      1: true, // true to true = unchanged
-      2: false, // true to false = changed
-      3: true, // false to true = changed
-      4: false, // false to false = unchanged
+    renderWithBrowserRouter(<DashboardConfigurationSubnetForm />, {
+      wrapperProps: { store },
     });
 
+    const checkboxes = screen.getAllByRole("checkbox");
+
+    await userEvent.click(checkboxes[1]);
+    await userEvent.click(checkboxes[2]);
```

Comment:
It might be better to target each checkbox by label instead of an index so that we don't rely on a particular order of elements (and the relationship between data and rendered form is more clear). I'm happy for you to leave this as is not to increase the scope of this migration work unnecessarily.

Reply:
Gonna leave this as is, from a brief bit of testing trying to select them by label is too large of a rabbit hole, since the aria-labelling on them doesn't appear to work properly (label comes out as [Object object]). Might be worth making another issue to fix thsi